coalib/settings/ConfigurationGathering.py

```diff
@@ -179,7 +182,8 @@ def warn_nonexistent_targets(targets, sections, log_printer=None):
 
     # Can't be summarized as python will evaluate conditions lazily, those
     # functions have intended side effects though.
-    files_config_absent = warn_config_absent(sections, 'files')
+    files_config_absent = warn_config_absent(
```

Comment:
all of the rest of this function is useless if `only_global_bears`, because that implicitly says there is at least one bear, so `bears_config_absent` can never be `True`.

Bail out early if `bears_config_absent` is True.

Reply:
i see, all of this function is useless, cause we dont require files and we already know that there is atleast one bear provided by the user if `only_global_bears` is true